pkg/virt-launcher/virtwrap/api/converter.go

```diff
@@ -494,6 +494,14 @@ func Convert_v1_VirtualMachine_To_api_Domain(vmi *v1.VirtualMachineInstance, dom
 	}
 
 	getInterfaceType := func(iface *v1.Interface) string {
+		if iface.Slirp != nil {
+			// Slirp configuration works only with e1000 or rtl8139
+			if iface.Model != "e1000" && iface.Model != "rtl8139" {
+				log.Log.Infof("The network interface type of %s was changed from virtio to e1000 due to unsupported interface type by qemu slirp network", iface.Name)
```

Comment:
I don't think the log message is now correct? It says it changed from virtio but there are other options.

Reply:
remove the virtio word